setup/README.md

```diff
@@ -0,0 +1,20 @@
+# Installation script
+
+This folder contains a script that is useful for the installation of the k3s cluster locally with just one command.
+
+The script will perform the following tasks:
+1. Create a management user (mgmt)
```

Comment:
Can we create this user by downloading the private key from some private website or something similar? Just to have something more secure (and comfortable) than a user with password "root".